fontforgeexe/cvruler.c

```diff
@@ -1003,6 +1003,5 @@ return;
 	    }
 	    prev_rect = rect;
 	}
-	RulerLingerMove(cv);	/* in case things are moving or scaling */
```

Comment:
Instead of calling this from the expose event, I call it in response to the charview doing a scroll or zoom.